### PR TITLE
consolidated items related to credentials to a single module.

### PR DESCRIPTION
**Pull request by jasonbrackman**
captive_http writes out valid credentials
captive_portal reads/deletes credentials

### CODE REVIEW

File: captive_portal.py [hunk 1]
```diff
@@ -138,7 +125,7 @@ def captive_portal(self):
 
                 if self.check_valid_wifi():
                     print("Connected to WiFi!")
-                    self.http_server.set_ip(self.local_ip, self.ssid)
+                    self.http_server.set_ip(self.local_ip, self.essid)
```
Comment (anson-vandoren) [inline captive_portal.py]: This will cause the ESP8266 SSID to be displayed to the user after connection, instead of the newly-connected AP SSID.

**Review by anson-vandoren — CHANGES_REQUESTED** [review]
The rest looks good, thanks!

Calling `self.http_server.set_ip()` needs to be with the SSID of the network we just connected to. `self.essid` is the SSID that the ESP8266 was broadcasting on to display the captive portal.

File: captive_portal.py (hunk 1 above)
Comment (anson-vandoren) [reply captive_portal.py]: @jasonbrackman I also added a CONTRIBUTORS.md file - feel free to add yourself to that list as well (with or without your GitHub username, or website, or whatever other details you do or don't want to add)

Comment (jasonbrackman) [reply captive_portal.py]: May take me a day or two -- have a few meetings that are eating into my evening time :).  But will update soon.

**Pull request merged**